elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/data/value/ElasticsearchExprValueFactory.java

```diff
@@ -77,6 +78,7 @@
       new DateTimeFormatterBuilder()
           .appendOptional(SQL_LITERAL_DATE_TIME_FORMAT)
           .appendOptional(STRICT_DATE_OPTIONAL_TIME_FORMATTER)
+              .appendOptional(STRICT_DATE_FORMATTER)
```

Comment:
Should `STRICT_HOUR_MINUTE_SECOND_FORMATTER` be registered here too?

Reply:
removed STRICT_DATE_FORMATTER as the issue was fixed with STRICT_HOUR_MINUTE_SECOND_FORMATTER.